astro/src/pages/index.astro

```diff
@@ -66,7 +83,10 @@ changeLanguage("en");
     <div class="listing"></div>
     <div class="blog"></div>
      -->
-  </div>
+  
 
 
 </BaseLayout>
+
+<script src="../../../node_modules/flowbite/dist/flowbite.js"></script>
```

Comment:
I think this is automatically injected by astro, is it necessary to add it this way?